sql/parse/parse_test.go

```diff
@@ -3237,6 +3237,10 @@ var fixturesErrors = map[string]*errors.Kind{
 	`SELECT a, count(i) over (partition by y) FROM foo`:         ErrUnsupportedFeature,
 	`SELECT i, row_number() over (order by a) group by 1`:       ErrUnsupportedFeature,
 	`SELECT i, row_number() over (order by a), max(b)`:          ErrUnsupportedFeature,
+	`SHOW COUNT(*) WARNINGS`:                                    ErrUnsupportedFeature,
```

Comment:
These now parse in vitess, but we still don't support them yet. Testing that `convertShow` handles things appropriately.